package.json

```diff
@@ -80,7 +80,7 @@
     "@types/ajv": "^1.0.0",
     "@types/chai": "^4.2.22",
     "@types/cors": "^2.8.12",
-    "@types/express": "^4.17.13",
+    "@types/express": "4.17.13",
```

Comment:
What's so glorious about this particular version? 🤔

Reply:
I think it was typing conflicts with m-a-b. Tedious. I'll see if I can stop pinning.